akka-actor/src/main/scala/akka/actor/Scheduler.scala

```diff
@@ -91,6 +96,9 @@ trait Scheduler {
    * If the `Runnable` throws an exception the repeated scheduling is aborted,
    * i.e. the function will not be invoked any more.
    *
+   * @throws IllegalArgumentException if the given delays exceed the maximum 
+   * reach (calculated as: `delay / tickNanos > Int.MaxValue`).
```

Comment:
Hm, can we report somehow the actual maximum value? It would be more robust than the user having to calculate it.